modules/preview-react/text/stories/examples/BodyText.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {BodyText as BodyTextComponent} from '@workday/canvas-kit-preview-react/text';
```

Comment:
What's your reasoning of renaming the component this way? just curious

Reply:
It's just to keep BodyText as storybook story name